drone/src/agent/executor.rs

```diff
@@ -220,7 +220,16 @@ impl<E: Engine> Executor<E> {
         let (send, mut recv) = channel(1);
         self.backend_to_listener
             .insert(spawn_request.backend_id.clone(), send);
-
+        self.backend_to_monitor.insert(
```

Comment:
Since `resume_backends` calls `run_backend`, we no longer need to insert this in `resume_backends` (lines 201-212 can be deleted)